sortinghat/core/api.py

```diff
@@ -246,3 +253,93 @@ def update_profile(uuid, **kwargs):
         raise InvalidValueError(msg=str(e))
 
     return uidentity
+
+
+@django.db.transaction.atomic
+def enroll(uuid, organization, from_date=None, to_date=None):
+    """Enroll a unique identity in an organization.
+
+    The function enrolls a unique identity, identified by `uuid`,
+    in the given `organization`. Both identity and organization must
+    exist before adding this enrollment to the registry. Otherwise,
+    a `NotFoundError` exception will be raised.
+
+    The period of the enrollment can be given with the parameters
+    `from_date` and `to_date`, where `from_date <= to_date`. Default
+    values for these dates are `1900-01-01` and `2100-01-01`.
+
+    Existing enrollments for the same unique identity and organization
+    which overlap with the new period will be merged into a single
+    enrollment.
+
+    If the given period for that enrollment is enclosed by one already
+    stored, the function will raise an `AlreadyExistsError` exception.
+
+    The unique identity object with updated enrollment data is returned
+    as the result of calling this function.
+
+    :param uuid: unique identifier
+    :param organization: name of the organization
+    :param from_date: date when the enrollment starts
+    :param to_date: date when the enrollment ends
+
+    :returns: a unique identity with enrollment data updated
+
+    :raises NotFoundError: when either `uuid` or `organization` are not
+        found in the registry.
+    :raises InvalidValeError: raised in three cases, when either identity or
+        organization are None or empty strings; when "from_date" < 1900-01-01 or
+        "to_date" > 2100-01-01; when "from_date > to_date"
+    :raises AlreadyExistsError: raised when given period for that enrollment
```

Comment:
typo -> missing the between when and given ?

Reply:
It should be fixed with the latest changes.